apis/apiextensions/v1/composition_transforms.go

```diff
@@ -73,6 +73,29 @@ type MathTransform struct {
 	Multiply *int64 `json:"multiply,omitempty"`
 }
 
+type customError struct {
+    err string
+}
+
+func (e customError) Error() string {
+    return e.err
+}
+
+// Resolve runs the Math transform.
+func (m *MathTransform) Resolve(input interface{}) (interface{}, error) {
+	if m.Multiply == nil {
+		return nil, customError{err:"no multiplier provided for math transform"}
+	}
+	switch i := input.(type) {
+	case int64:
+		return float64(*m.Multiply) * float64(i), nil
+	case int:
+		return float64(*m.Multiply) * float64(i), nil
+	default:
+		return nil, customError{err:"input for math transform is not a number"}
```

Comment:
Please comply with the error reporting consistent with the rest of the codebase.  Creating a custom error for a single function looks suboptimal.

Reply:
Thanks, I refered from other files and made those changes

https://github.com/Vrashabh-Sontakke/crossplane/blob/c9d66544c2fe3924036e76126618b9a94504eaf1/apis/apiextensions/v1/composition_transforms.go#L77-L95